src/libertem/web/server.py

```diff
@@ -229,13 +233,24 @@ def run(host, port, browser, local_directory, numeric_level, token, preload, str
         port = _port
 
     main(bound_sockets, event_registry, shared_state, token)
+    if executor_spec is not None:
+        # Must happen after main() so that Dask uses our event loop
+        shared_state.add_executor(executor_spec)
 
     try:
         is_ipv6 = isinstance(ipaddress.ip_address(host), ipaddress.IPv6Address)
     except ValueError:
         is_ipv6 = False
     url = f'http://[{host}]:{port}' if is_ipv6 else f'http://{host}:{port}'
-    log.info(f"listening on {url}")
+    if open_ds is not None:
+        url = f'{url}/#action=open&path={open_ds}'
+    hostname = socket.gethostname()
+    msg = f"""
+
+LiberTEM listening on {url}
+                      {url.replace(host, hostname)}
```

Comment:
It might make more sense to *only* show this line if we are listening on `0.0.0.0` or `::`, because presumably if the user specifies something else for `host` they will be able to use the normal link text which already has their supplied IP / name inserted...

Reply:
:+1:

Reply:
:+1: